README.md

```diff
@@ -207,6 +207,9 @@ cargo run -- build world
 # Builds a local package you have at <SOURCE> and adds it to the database.
 # The package does not have to be on crates.io.
 # The package must be on the local filesystem, git urls are not allowed.
+# Usually this command can be applied directly to a crate root
+# In certain scenarios it might be necessary to first package the respective
```

Comment:
Could you add some examples here which cases might need `cargo package`? 

Also, how the `cargo package` command would relate to the local build command itself?

Reply:
I wonder if we could have building a local crate package it then build from the package directly. Might need some minor rustwide changes to support.

Reply:
I think the most important thing would be make it really obvious how exactly packages are built. I do not understand this either at the moment. Just running
```
cargo +nightly rustdoc -- with the flags you think you need
```
is not identical to the build process but obviously the local build specified in the README also is (currently) not.